draft-ietf-masque-connect-ip-dns.md

```diff
@@ -57,39 +57,53 @@ informative:
+configuration details such as DNS information or IPv6 NAT64 prefixes (PREF64).
+In contrast, most existing VPN protocols provide mechanisms to exchange
+such configuration information.
+
+This document defines extensions that convey DNS and PREF64 configuration
+using HTTP capsules. This mechanism supports encrypted DNS transports
+and can be used to inform peers about network translation prefixes for IPv6/IPv4
+synthesis.
 
 --- middle
 
 # Introduction
 
 Proxying IP in HTTP ({{!CONNECT-IP=RFC9484}}) allows building a VPN through
-HTTP load balancers. However, at the time of writing, that mechanism doesn't
-offer a mechanism for communicating DNS configuration information inline. In
-contrast, most existing VPN protocols provide a mechanism to exchange DNS
-configuration information (e.g., {{?IKEv2=RFC7296}}). This document describes
-an extension that exchanges this information using HTTP capsules
-({{!HTTP-DGRAM=RFC9297}}). This document does not define any new ways to convey
-DNS queries or responses, only a mechanism to exchange DNS configuration
-information.
-
-Note that this extension is meant for cases where connect-ip is used like a
-Remote Access VPN (see {{Section 8.1 of CONNECT-IP}}) or Site-to-Site VPN
+HTTP load balancers. However, at the time of writing, that mechanism lacks
+a way to communicate network configuration details such as DNS information
+or IPv6 NAT64 prefixes (PREF64). In contrast, most existing VPN protocols provide
+mechanisms to exchange such configuration information (for example
+{{?IKEv2=RFC7296}} supports discovery of DNS servers).
+
+This document defines extensions that allow CONNECT-IP peers to convey DNS
+and PREF64 configuration information to its peer using HTTP capsules
+({{!HTTP-DGRAM=RFC9297}}). These extensions do not define any new way to
+transport DNS queries or responses, but instead enable the exchange of DNS
+resolver configuration and NAT64 prefix information inline with CONNECT-IP
+sessions.
+
+Note that these extensions are meant for cases where CONNECT-IP operates as a
+Remote Access VPN (see {{Section 8.1 of CONNECT-IP}}) or a Site-to-Site VPN
 (see {{Section 8.2 of CONNECT-IP}}), but not for cases like IP Flow Forwarding
 (see {{Section 8.3 of CONNECT-IP}}).
 
-This specification uses Service Bindings ({{!SVCB=RFC9460}}) to exchange
-information about nameservers, such as which encrypted DNS transport is
+For DNS configuration, this specification uses Service Bindings ({{!SVCB=RFC9460}})
+to exchange information about nameservers, such as which encrypted DNS transport is
 supported. This allows support for DNS over HTTPS ({{!DoH=RFC8484}}), DNS over
 QUIC ({{!DoQ=RFC9250}}), DNS over TLS ({{!DoT=RFC7858}}), unencrypted DNS over
 UDP port 53 and TCP port 53 ({{!DNS=RFC1035}}), as well as potential future DNS
-transports.
+transports. PREF64 configuration is represented in a way similar to Router
+Advertisement option defined in {{Section 4 of !PREF64-RA=RFC8781}}.
```

Comment:
nit since this is just an explanation and not a dependency, it should be an informative reference

```suggestion
Advertisement option defined in {{Section 4 of ?PREF64-RA=RFC8781}}.
```